### PR TITLE
[Feature](executor)Support ShowProcessStmt Show all Fe connection

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

Support ShowProcessStmt Show all Fe connection.
```
set show_all_fe_connection = true;

    mysql [(none)]>show processlist;
+------------------+------+------+------------------+---------------------+----------+------+---------+------+-------+-----------------------------------+------------------+------------+
| CurrentConnected | Id   | User | Host             | LoginTime           | Catalog  | Db   | Command | Time | State | QueryId                           | Info             | FE         |
+------------------+------+------+------------------+---------------------+----------+------+---------+------+-------+-----------------------------------+------------------+------------+
| Yes              |    0 | root | 127.0.0.1:52494 | 2024-02-07 12:08:01 | internal |      | Query   |    0 | OK    | 2fa7a4724ccc4090-b794ff24504ad20f | show processlist | 127.0.0.1|
+------------------+------+------+------------------+---------------------+----------+------+---------+------+-------+-----------------------------------+------------------+------------+
1 row in set (1.75 sec)

set show_all_fe_connection = false;
mysql [(none)]>show processlist;
+------------------+------+------+------------------+---------------------+----------+------+---------+------+-------+-----------------------------------+------------------+
| CurrentConnected | Id   | User | Host             | LoginTime           | Catalog  | Db   | Command | Time | State | QueryId                           | Info             |
+------------------+------+------+------------------+---------------------+----------+------+---------+------+-------+-----------------------------------+------------------+
| Yes              |    0 | root | 127.0.0.1:52494 | 2024-02-07 12:08:01 | internal |      | Query   |    0 | OK    | a84ce3be2c464042-80cfced4936bb8b2 | show processlist |
+------------------+------+------+------------------+---------------------+----------+------+---------+------+-------+-----------------------------------+------------------+
1 row in set (0.00 sec)

```

### CODE REVIEW

**Review by yiguolei — DISMISSED** [review]

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java [hunk 1]
```diff
@@ -456,11 +461,52 @@ private void handleShowProcesslist() {
...
+        }
+
+        if (isShowAllFe) {
+            try {
+                TShowProcessListRequest request = new TShowProcessListRequest();
+                request.setShowFullSql(isShowFullSql);
+                List<Pair<String, Integer>> frontends = FrontendsProcNode.getFrontendWithRpcPort(Env.getCurrentEnv(),
+                        false);
+                FrontendService.Client client = null;
+                for (Pair<String, Integer> fe : frontends) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java:481]: Skip master FE?

**Review by morningman — COMMENTED** [review]
Better to add a regression test case.
Although there is only one FE node in regression test env, but you can also check the column list returned

**Review by morningman — COMMENTED** [review]
Missing document

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**